tests/nightly/test_np_large_array.py

```diff
@@ -1886,3 +1885,36 @@ def test_array_split():
     assert out[0][0][0] == 0
     assert out[1][-1][-1] == 2
 
+
+@use_np
+def test_std():
+    N = 2*20
+    inp = np.zeros((2, INT_OVERFLOW))
+    inp[-1, -1] = N
+    inp.attach_grad()
+    with mx.autograd.record():
+        out = np.std(inp, axis=1)
+        out.backward()
+    assert out.shape == (2, )
```

Comment:
can we directly comapre the outputs with actual numpy's operators ? call `import numpy as _np`

Reply:
I tried that first but it would take too long to run on large tensors so I instead derived a analytical formula. The correctness has been verified with small tensor size first then I switched to large tenosr

Reply:
https://www.mathsisfun.com/data/standard-deviation-formulas.html